sdcm/cluster.py

```diff
@@ -2262,6 +2270,22 @@ def restart_scylla(self, nodes=None):
             node.start_scylla(verify_up=True)
             self.log.debug("'{0.name}' restarted.".format(node))
 
+    def gather_additional_information(self, storing_dir):
```

Comment:
`gather_additional_information` not much informative. Which information? Why to gather it? Why is it additional?  In code we collect/get/retrieve data or logs. When giving names to the method ask yourself, if somebody that is new to the code and sees only this method, can he understand what it does without reading method implementation?
Change to smth. like `gather_additional_information` ->  `collect_logs`

Reply:
renamed